### ISSUE TITLE
Question: Local Password Save

### ISSUE THREAD

**Issue by djtdoug** [StuartStephenKing/TurboModelerInstaller#1]
Hi Stuart,  Love the progress on Turbo Modeler.   Is there a way for me to locally save me credentials?  The default that gets populated appears to be your fopmom environment.   I'd like to work with it on fopmqc2.   Thanks.

**Comment by StuartStephenKing** [comment]
Saving credentials on the client isn't really a great idea for security reasons.

I'm currently defaulting admin/apple as the username and password just to make it easier to quicker to perform manual testing (less clicks).  I plan to remove the default username and password so the login behaviour is exactly like Architect/Perspectives.

Worth noting that the tool also support CAM authentication exactly like Architect/Perspectives.  No support for IntegratedSecurityMode 3 at the moment.

**Comment by djtdoug** [comment]
Fair enough....I understand the security issue.  It's really fun to play around with.  I'm sure you have a long list of things to do.  I'd vote for being able to see Aliases for the elements.
